core/components/molecules/sidesheet/__stories__/index.story.jsx

```diff
@@ -1,58 +1,52 @@
 import * as React from 'react';
 import { action } from '@/utils/action';
-import { Text, Sidesheet, Button, ModalDescription } from '@/index';
+import { Sidesheet, Button, Label, Text } from '@/index';
 
 export const all = () => {
   const [open, setOpen] = React.useState(false);
-  const seperator = false;
-  const backIcon = false;
-  const stickFooter = false;
-  const backdropClose = false;
-  const dimension = 'regular';
 
   const onClose = () => {
     setOpen(false);
     action('on close triggered')();
   };
 
-  const backIconCallback = () => {
-    action('back icon clicked')();
-  };
-
   const headerOptions = {
-    backIcon,
-    backIconCallback: backIcon ? backIconCallback : undefined,
     heading: 'Heading',
     subHeading: 'Subheading',
   };
 
   const options = {
     onClose,
     open,
-    dimension,
-    seperator,
     headerOptions,
-    stickFooter,
-    backdropClose,
     footer: (
       <>
         <Button appearance="primary" className="mr-4">
           Primary
         </Button>
-        <Button appearance="basic">Basic</Button>
+        <Button appearance="basic">Cancel</Button>
       </>
     ),
   };
 
-  const modalDescriptionOptions = {
-    title: 'Description Title',
+  const SidesheetDescription = (params) => {
+    const { label, description } = params;
+    return (
+      <div className="pt-4 pb-4">
+        {label && <Label withInput={true}>{label}</Label>}
+        {label && description && <br />}
+        {description && <Text>{description}</Text>}
+      </div>
```

Comment:
Please use the className `py-4` Also remove the conditional use of `<br />`

Reply:
py-4 is included, but conditional use of <br/> is needed for component reuse.